### ISSUE TITLE
Main Page "feature boxes" fail accessibility on dark mode using Chrome.

### ISSUE THREAD

**Issue by rowan-austin** [powerstackdev/power-stack#71]
**Describe the bug**
On the main homepage https://powerstack.dev/ the "feature boxes" have a very light colour text when compared to the background, this is occurring on Chrome using dark mode.

**To Reproduce**
1. Go to the homepage
2. Switch Chrome to be dark mode
3. Observe boxes

**Expected behavior**
The boxes' text passes accessibility. 

**Desktop (please complete the following information):**
 - OS: MacOS Monterey 12.4
 - Browser: Chrome
 - Version: 105.0.5195.102
